tbtadm/controller.cpp

```diff
@@ -74,9 +74,31 @@ const std::string opt_remove      = "remove";
 const std::string opt_remove_all  = "remove-all";
 const std::string opt_once_flag   = "--once";
 
-const std::string indent     = "|   ";
+const std::string indent     = "│   ";
+const size_t indentLength    = 4;
 const std::string indentLast = "    ";
 
+enum {
```

Comment:
`enum class`? (more type-safe)

Reply:
Ah, no, you use it below for array index so this will require an explicit cast later.
Your choice.